day_6/exercises/person.js

```diff
@@ -7,5 +7,20 @@ Create a person class with at least 2 properties and 2 behaviors.  Call all
 person methods below the class so that they print their result to the
 terminal.
 */
+class Person {
```

Comment:
For this class of `Person`, you are passing arguments into the constructor method and designating them as attributes correctly, but I must say, the attributes and methods don't make much sense in context.  What do you think of having attributes like `name`, `age`, `gender`, `favoriteSong`, `favoriteColor`, etc. for the Person class as well as methods that could update or modify those attributes?

Reply:
I've updated the class :)